src/antigravity/agent.py

```
@@ -0,0 +1,376 @@
+Implements the Think-Act-Reflect loop with MCP integration.
+"""
+
+import json
+import time
+import os
+import sys
+import asyncio
+import inspect
+import importlib.util
+from pathlib import Path
+from typing import Any, Callable, Dict, List, Optional, Tuple
```

Ruff (0.14.8)

Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)